src/icons/IntercomIcon.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
```

Comment:
All of the Icon files will be refactored, the changes on them below are to remove dependencies on deleted code